src/components/tx/AdvancedParams/AdvancedParamsForm.tsx

```diff
@@ -84,11 +84,13 @@ const AdvancedParamsForm = ({ params, ...props }: AdvancedParamsFormProps) => {
         <form onSubmit={onFormSubmit}>
           <DialogContent>
             <Grid container spacing={2}>
-              <Grid item xs={12}>
-                <Typography variant="body1" fontWeight={700}>
-                  Safe transaction
-                </Typography>
-              </Grid>
+              {(params.nonce !== undefined || !!params.safeTxGas) && (
```

Comment:
Added this condition so that the title is only shown if there is either a nonce or safeTxGas field